google-cloud-pubsub/lib/google/cloud/pubsub/service.rb

```diff
@@ -297,6 +329,85 @@ def seek subscription, time_or_snapshot
           end
         end
 
+        ##
+        # Lists schemas in the current (or given) project.
+        # @param view [String, Symbol, nil] Possible values:
+        #   * `BASIC` - Include the name and type of the schema, but not the definition.
+        #   * `FULL` - Include all Schema object fields.
+        #
+        def list_schemas view, options = {}
+          schema_view = Google::Cloud::PubSub::V1::SchemaView.const_get view.to_s.upcase
+          paged_enum = schemas.list_schemas parent:     project_path(options),
+                                            view:       schema_view,
+                                            page_size:  options[:max],
+                                            page_token: options[:token]
+
+          paged_enum.response
+        end
+
+        ##
+        # Creates a schema in the current (or given) project.
+        def create_schema schema_id, type, definition, project: nil
```

Comment:
why do we have `options` hashes in some methods, and keyword arguments in others?

Reply:
That was an accident. And not the first. (`create_snapshot subscription, snapshot_name, labels: nil`). I would love to convert the library entirely to keyword args, but not in this PR. Will change this to `options`.